src/NextCloud.py

```diff
@@ -0,0 +1,18 @@
+from .requester import Requester
+from .api_wrappers import API_CLASSES
+
+
+class NextCloud(object):
+
+    def __init__(self, endpoint, user, password, js=False):
```

Comment:
Please rename `js` to `json_output`, or introduce notion output types XML or JSON. The original `js` is not really descriptive.